rollup.config.js

```diff
@@ -5,10 +5,11 @@ import replace from '@rollup/plugin-replace';
 
 /** @type {import("rollup").RollupOptions} */
 const defaultConfig = {
-  input: 'src/index.ts',
+  input: ['src/index.ts', 'src/react.ts'],
```

Comment:
q: any special reason why you'd like to keep using `preserveModules`?

Reply:
Compared to a single-file ESM build, tree shaking seems to be working a LOT better.

With `preserveModules` (0.2.0):
![image](https://user-images.githubusercontent.com/4282439/103455203-704c3480-4ceb-11eb-9a30-cedac240a6e9.png)
Normal rollup build (0.1.0):
![image](https://user-images.githubusercontent.com/4282439/103455211-83f79b00-4ceb-11eb-804b-705c362897b0.png)

Reply:
Was expecting this answer and it's a good one. Just a note for completeness - this doesn't quite improve tree-shaking (given my own definition of what tree-shaking is). It allows bundlers to leverage `"sideEffects": false` - but this flag is a hack around tree-shaking as ideal tree-shaking in practice is sometimes hard to achieve (due to initialization side-effects that include function calls and property accesses). 

Both things are not mutually exclusive though as `"sideEffects": false` relies heavily on the file boundaries that you have set up as an author which might not produce 100% ideal results either.